docs/en/observability/monitor-uptime-synthetics.asciidoc

```diff
@@ -5,9 +5,20 @@
 <titleabbrev>Synthetic monitoring</titleabbrev>
 ++++
 
-The {uptime-app} in {kib} lets you monitor user journeys and full page loads using our browser monitor,
-and also monitor the availability and response times of applications and services using our lightweight HTTP/S,
-TCP, and ICMP monitors.
+Synthetic monitors go beyond classic end-to-end tests and monitoring practices to ensure that your software continues to work as expected in many conditions. 
```

Comment:
IMO the words `go beyond` and `many conditions` blurry the meaning we'd like to convey.

For more preciseness, I'd probably say that:

> Synthetic monitoring extends traditional end-to-end testing techniques because it allows your tests to run continuously on the cloud.
> With Elastic's Synthetic Monitoring product, you can reuse the same journeys that you've used to validate the software on your machine to assert that your application continues to work after a deployment.